sanity.functional JCL_Test_SE80 test_RevealDirect_CrackCallerSensitive Unable to read java.lang.invoke.SecurityFrame.class bytes

https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/28
JCL_Test_SE80_*
JCL_TEST_Java-Lang-Invoke
test_RevealDirect_CrackCallerSensitive
```
FAILED: test_RevealDirect_CrackCallerSensitive
java.lang.Error: Unable to read java.lang.invoke.SecurityFrame.class bytes
	at java.lang.invoke.SecurityFrameInjector$1.run(SecurityFrameInjector.java:103) from /home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdkbinary/j2sdk-image/jre/lib/rt.jar
	at java.lang.invoke.SecurityFrameInjector$1.run(SecurityFrameInjector.java:96) from /home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdkbinary/j2sdk-image/jre/lib/rt.jar
	at java.security.AccessController.doPrivileged(AccessController.java:678) from /home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdkbinary/j2sdk-image/jre/lib/rt.jar
	at java.lang.invoke.SecurityFrameInjector.initializeSecurityFrameClassBytes(SecurityFrameInjector.java:96) from /home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdkbinary/j2sdk-image/jre/lib/rt.jar
	at java.lang.invoke.SecurityFrameInjector.wrapHandleWithInjectedSecurityFrame(SecurityFrameInjector.java:129) from /home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdkbinary/j2sdk-image/jre/lib/rt.jar
	at java.lang.invoke.SecurityFrameInjector.wrapHandleWithInjectedSecurityFrameIfRequired(SecurityFrameInjector.java:207) from /home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdkbinary/j2sdk-image/jre/lib/rt.jar
	at java.lang.invoke.MethodHandles$Lookup.findStatic(MethodHandles.java:547) from /home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdkbinary/j2sdk-image/jre/lib/rt.jar
	at org.openj9.test.java.lang.invoke.Test_MethodHandleInfo.test_RevealDirect_CrackCallerSensitive(Test_MethodHandleInfo.java:789) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/jvmtest/functional/Java8andUp/GeneralTest.jar)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) from /home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdkbinary/j2sdk-image/jre/lib/rt.jar
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) from /home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdkbinary/j2sdk-image/jre/lib/rt.jar
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) from /home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdkbinary/j2sdk-image/jre/lib/rt.jar
	at java.lang.reflect.Method.invoke(Method.java:498) from /home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdkbinary/j2sdk-image/jre/lib/rt.jar
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.TestRunner.privateRun(TestRunner.java:648) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.TestRunner.run(TestRunner.java:505) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.TestNG.runSuites(TestNG.java:1049) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.TestNG.run(TestNG.java:1017) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.TestNG.privateMain(TestNG.java:1354) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
	at org.testng.TestNG.main(TestNG.java:1323) from sun.misc.Launcher$AppClassLoader@eff23263(file:/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdk-tests/TKG/lib/testng.jar)
Caused by: java.io.IOException: length cannot be negative: -1
	at sun.misc.IOUtils.readFully(IOUtils.java:305) from /home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdkbinary/j2sdk-image/jre/lib/rt.jar
	at java.lang.invoke.SecurityFrameInjector$1.run(SecurityFrameInjector.java:100) from /home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_s390x_linux_OpenJDK8/openjdkbinary/j2sdk-image/jre/lib/rt.jar
	... 31 more
```

--- a/jcl/src/java.base/share/classes/java/lang/invoke/SecurityFrameInjector.java
+++ b/jcl/src/java.base/share/classes/java/lang/invoke/SecurityFrameInjector.java
@@ -1,6 +1,6 @@
 /*[INCLUDE-IF Sidecar17]*/
 /*******************************************************************************
- * Copyright (c) 2012, 2019 IBM Corp. and others
+ * Copyright (c) 2012, 2020 IBM Corp. and others
  *
  * This program and the accompanying materials are made available under
  * the terms of the Eclipse Public License 2.0 which accompanies this
@@ -106,7 +106,7 @@ final class SecurityFrameInjector {
 								return Lookup.class.getResourceAsStream("/java/lang/invoke/SecurityFrame.class").readAllBytes(); //$NON-NLS-1$
 								/*[ELSE]*/
 								InputStream is = Lookup.class.getResourceAsStream("/java/lang/invoke/SecurityFrame.class"); //$NON-NLS-1$
-								return IOUtils.readFully(is, -1, true);
+								return IOUtils.readAllBytes(is);
 								/*[ENDIF]*/
 							} catch(java.io.IOException e) {
 								/*[MSG "K056A", "Unable to read java.lang.invoke.SecurityFrame.class bytes"]*/

